Client/Frontend/Widgets/Menu/MenuView.swift

```diff
@@ -400,6 +400,14 @@ extension MenuView: UICollectionViewDelegateFlowLayout {
     func collectionView(collectionView: UICollectionView, didSelectItemAtIndexPath indexPath: NSIndexPath) {
         menuItemDelegate?.menuView(self, didSelectItemAtIndexPath: indexPath)
     }
+    
+    func collectionView(collectionView: UICollectionView, shouldSelectItemAtIndexPath indexPath: NSIndexPath) -> Bool {
+        guard let x = menuItemDelegate?.menuView(self, shouldSelectItemAtIndexPath: indexPath) else {
```

Comment:
I would prefer a more descriptive variable name here, rather than `x`. Perhaps `shouldSelect` or `canBeSelected` or something?

also could be made terser like this

` return menuItemDelegate?.menuView(self, shouldSelectItemAtIndexPath: indexPath) ?? false`